Cubical/Algebra/CommAlgebra/FPAlgebra.agda

```diff
@@ -104,21 +104,16 @@ module _ {R : CommRing ℓ} where
                            (incInIdeal (Polynomials n) relation i ) ⟩
         0a (snd FPAlgebra) ∎
 
-      inducedHom :
+      module _
         (A : CommAlgebra R ℓ)
         (values : FinVec ⟨ A ⟩ n)
         (relationsHold : (i : Fin m) → evPoly A (relation i) values ≡ 0a (snd A))
-        → CommAlgebraHom FPAlgebra A
-      inducedHom A values relationsHold =
-        quotientInducedHom
-          (Polynomials n)
-          relationsIdeal
-          A
-          freeHom
-          isInKernel
-        where
+        where abstract
```

Comment:
Is the 'abstract'-block inside an 'abstract' block needed due to some weirdness of how 'abstract' works, or is it needed for making the inside definitions abstract to the other abstract definitions?
I guess a comment would be good here.